spec/javascripts/packages/step-indicator/index-spec.js

```diff
@@ -1,35 +1,31 @@
 import StepIndicator from '@18f/identity-step-indicator';
 import userEvent from '@testing-library/user-event';
 import { useSandbox } from '../../support/sinon';
+import useDefineProperty from '../../support/define-property';
 
 describe('StepIndicator', () => {
   const sandbox = useSandbox();
+  const defineProperty = useDefineProperty();
 
-  beforeEach(() => {
+  function initialize({ currentStepIndex = 0 } = {}) {
     document.body.innerHTML = `
       <div role="region" aria-label="Step progress" class="step-indicator">
         <ol class="step-indicator__scroller">
-          <li class="step-indicator__step">
-            <span class="step-indicator__step-title">
-              Step one
-            </span>
-            <span class="usa-sr-only">
-              Completed
-            </span>
-          </li>
-          <li class="step-indicator__step step-indicator__step--current">
-            <span class="step-indicator__step-title">
-              Step two
-            </span>
-            <span class="usa-sr-only">
-              Current step
-            </span>
-          </li>
+          ${Array.from(Array(5))
```

Comment:
Javascript wishes it had `5.times.map` built in 😭

Reply:
😄  [Array#fill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill) is close, but sadly only supports static fill values.